roles/openshift_register_nodes/library/kubernetes_register_node.py

```diff
@@ -255,7 +255,7 @@ def exists(self):
         return False
 
     def create(self):
-        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', 'node', '-f', '-']
+        cmd = ['/usr/bin/osc'] + self.client_opts + ['create', '-f', '-']
```

Comment:
@jwhonce latest commit adds a parameter to the kubernetes_register_node module to allow overriding the default command used (which is kubectl).